xml/security_ldap_modules.xml

```diff
@@ -15,13 +15,13 @@
  also scoping, https://directory.fedoraproject.org/docs/389ds/design/memberof-scoping.html
  cjs, March 12 2022-->
  <info>
-  <title>Managing modules</title>
+  <title>Managing plug-ins</title>
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
  </info>
   <para>
-    Use the following command to list all available modules, enabled and
+    Use the following command to list all available pug-ins, enabled and
```

Comment:
pug-ins is a typo :) should be "plugins"

Reply:
Oops, thanks, changed to 'plug-ins' now.